### PR TITLE
Feedback on workshop docs.

### PR DESCRIPTION
**Pull request by siggy**
This PR is all TIOLI, edit/merge as you like.

Signed-off-by: Andrew Seigner <siggy@buoyant.io>

I think one particularly compelling change is to avoid tearing down the demos after edits. It allows Grafana and Prometheus to stay up, and gives a pretty sweet picture of the change in stats:

screen shot 2018-03-07 at 10 29 13 am

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+### Clone repo
+
+```bash
+git clone https://github.com/adleong/qcon-linkerd-workshop.git
+```
+
+## Further reading
+
+Linkerd provides many more features. For further reading, head over to
+[linkerd.io](https://linkerd.io). For more examples, check out the
```
Comment (rmars) [inline README.md]: I would maybe even link straight to linkerd.io/features so they can jump right in to reading!

File: tls/linkerd-client.yml [hunk 2]
```diff
@@ -15,5 +15,5 @@ routers:
     # initiate TLS on all outgoing connections it creates.
     tls:
       trustCerts:
-      - # FIXME
-      commonName: # FIXME
+      - /ca-chain.cert.pem
```
Comment (rmars) [inline tls/linkerd-client.yml]: I kind of like the `#FIXME`s here to indicate where the user should edit the configs. And if the filled out sections are in the READMEs they don't also need to be filled in here?

**Review by rmars — COMMENTED** [review]
oooo, I like these suggestions.

File: tls/linkerd-client.yml (hunk 2 above)
Comment (siggy) [reply tls/linkerd-client.yml]: ah! you're right, i did not mean to make these edits

**Comment by adleong** [comment]
Thank you @siggy and @rmars!  Really great edits and additions here!  :taco:

**Pull request merged**